modules/ROOT/pages/migration-cheat-sheet.adoc

```diff
@@ -5,30 +5,30 @@ endif::[]
 
 // author: Mariano Gonzalez
 
-To help you move from Mule 3 to Mule 4, we built this index to help you find the Mule 4 equivalent of the most common Mule 3 use cases.
+To help you move from Mule runtime engine (Mule) 3 to Mule 4, use this list to help you find the Mule 4 equivalent of the most common Mule 3 use cases.
 
-*IMPORTANT:* Keep in mind that this index is only intended to act as a quick entry to the most common migration scenarios. It doesn't contain the entire migration guide. For the full set of migration articles, please visit our Mule xref:index-migration.adoc[Mule 4 for Mule 3 Users] page.
+*IMPORTANT:* This list is a quick reference for the most common migration scenarios. It doesn't contain the entire migration guide. For the full set of migration articles, please visit xref:index-migration.adoc[Mule 4 for Mule 3 Users].
 
-* xref:configuring-properties.adoc[Configuration Properties]: There's a new way of handling configuration properties in Mule 4
-* xref:migration-secure-properties-placeholder.adoc[Using Secure Properties]: Secure placeholders are now a Runtime feature and no longer part of the Security Module
+* xref:configuring-properties.adoc[Configuration Properties]: There's a new way of handling configuration properties in Mule 4.
+* xref:migration-secure-properties-placeholder.adoc[Using Secure Properties]: Secure placeholders are now a Runtime feature and no longer part of the Security Module.
```

Comment:
Line 13 uses "Runtime" by itself. Should it be Mule or Mule runtime engine? Also not sure if security module needs caps. But caps confuse me.

Reply:
good catch! Should be Mule. Security module (they cap all module & connector names)